### PR TITLE
[DISCUSSION NO MERGE] New beneficial effects for viro2

### PR DESCRIPTION
**Pull request by Angelite191**
Okay, my life is getting a little cluttered at the moment and I'm having a hard time finishing this as I'm not a superb coder. But here it is. Please take the time to proof read, discuss and even pitch ideas.

-Bone fixing effect : Fixes broken bones and internal bleeding over a short time. Stage 4
-Pain resistance : exactly what it says on the tin. Stage 3
-Food Heal : Heals all damage except oxygen when nutriment is in the system. Stage 3
-Tox convert : converts toxins damage into brute and burn. Stage 2
-Dam convert : converts brute and burn damage into tox. Stage 2
-Blood stabilizer : Produces blood in the blood vessels when blood loss occurs. Stage 2

More are planned, including;
Stun nullify : removes weaken with a chance per tick. (will probably lean around 60% per tick). Stage 4

If my skill allowed it, I'd make some effects that change the infection rate and type, but I'll need to be a little less burnt out to achieve that. 

Anyway, discuss, have fun, call me an autist, reach a conclusion.

### CODE REVIEW

**Comment by ririchiyo** [comment]
Virology needs to be reworked at some point, because any "beneficial" virus is immediately deadly to the ability for it to rapidly mutate with any exposure to radiation either from the singularity, random event, or via other chemicals.

**Comment by Angelite191** [comment]
I can fix that. I saw that bit in the code. I can substantially reduce the chance of it mutating, OOOOOR, add a variable for "mutation resistance" so the virologists have more to do than just rolling a fucking dice.

**Comment by Fox-McCloud** [comment]
@ririchiyo 

Not to mention the fever effects.

**Comment by Arthurdentist** [comment]
more beneficial symptoms would be useful for the time being. I approve of this.

**Comment by Angelite191** [comment]
\>That feel when N3X15-sempai finally notices you
Right away!

**Comment by Arthurdentist** [comment]
as a note, I feel the most annoying thing is the lack of final stage symptoms that are neutral or beneficial. If you had a couple more stage four symptoms I feel that would cut out a lot of the busy work.

**Comment by Duny-** [comment]
bluethumbsup

**Comment by Angelite191** [comment]
Uuuuh, I'm having issues with compiling code right now and coderbus is silent, Can't progress until it's fixed.


screenshot_6

File: baystation12.dme [hunk 1]
```diff
@@ -946,7 +946,10 @@
 #include "code\modules\clothing\under\miscellaneous.dm"
 #include "code\modules\clothing\under\shorts.dm"
 #include "code\modules\clothing\under\syndicate.dm"
-#include "code\modules\clothing\under\ties.dm"
+#include "code\modules\clothing\under\accessories\accessory.dm"
```
Comment (ComicIronic) [inline baystation12.dme]: You shouldn't have these changes in your PR.
You need to rebase onto Bleeding-Edge.

**Comment by Angelite191** [comment]
Uuuuh, sorry about the commits, I won't rebase while still working on it again.

**Pull request closed without merging**